src/components/Controls.tsx

```diff
@@ -14,14 +13,24 @@ import {
   dropdownCss,
 } from '../styles';
 import type { State } from '../types';
+import Dropdown from './Dropdown';
+import Icon from './Icon';
 
 type ControlsProps = {
   state: State;
   setState: React.Dispatch<React.SetStateAction<State>>;
   mapbox?: string;
+  setSearchParams: SetURLSearchParams;
+  searchParams: URLSearchParams;
```

Comment:
let's not prop drill these and instead do this below
```
const [searchParams, setSearchParams] = useSearchParams();
```

Reply:
agreed, I had some worry about creating mutable copies of the same variable, but since they're reference types doesn't matter.